tests/unit/sstring_test.cc

```diff
@@ -127,12 +126,12 @@ BOOST_AUTO_TEST_CASE(test_str_starts_with) {
 }
 
 BOOST_AUTO_TEST_CASE(test_str_ends_with) {
-    BOOST_CHECK(std::string("abcdefg").ends_with("abcdefg"));
     BOOST_CHECK(sstring("abcdefg").ends_with("abcdefg"));
     BOOST_CHECK(sstring("abcdefg").ends_with("efg"sv));
     BOOST_CHECK(sstring("abcde").ends_with('e'));
     BOOST_CHECK(sstring("abcde").ends_with("de"));
     BOOST_CHECK(sstring("abcdefg").ends_with(""));
+    BOOST_CHECK(sstring("abcde").ends_with("abcde"));
```

Comment:
What's the point of adding this test when you have the same test with a different string a few lines up?

Reply:
sorry, i had a local change to address the same issue. but failed to remove it when creating this branch.